src/DotNetWorker/RpcExtensions.cs

```diff
@@ -10,12 +10,12 @@
 using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Azure.Functions.Worker.Invocation;
 
-namespace Microsoft.Azure.Functions.Worker
+namespace Microsoft.Azure.Functions.Worker.Rpc
```

Comment:
Moved this class to a more specific namespace as the `**Rpc` methods were polluting the entire project.

Reply:
Good call.